sdk/storage/Azure.Storage.Common/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Fixed bug to ensure that the accumulated disposables within the internal `StorageWriteStream` are disposed properly in the event that an exception is thrown during the disposal process. (#47781)
 - Fixed bug to Redact SAS credentials from Error.SasCredentialRequiresUriWithoutSas message.
 - Fixed bug where LazyLoadingReadOnlyStream overprovisions the default buffer memory for OpenRead
+- Fixed bug to implement IAsyncDisposable in StorageWriteStream
```

Comment:
Not sure if i'd call this a bug or a feature. There was no behavior that was fixed; we just now offer an additional behavior.